examples/jn516x/tsch/tx-power-verification/node/Makefile

```diff
@@ -0,0 +1,25 @@
+CONTIKI_PROJECT = node
+
+TARGET ?= jn516x
+JN516x_WITH_DONGLE = 1
+
+CONTIKI=../../../../..
+CONTIKI_WITH_IPV6 = 1
+
+CFLAGS += -DPROJECT_CONF_H=\"project-conf.h\"
+PROJECTDIRS += .. ../../tools
+PROJECT_SOURCEFILES += rich.c 
```

Comment:
I would suggest renaming rich (no meaning for contiki upstream) to rpl-tools, as it was done for https://github.com/contiki-os/contiki/tree/master/examples/jn516x/rpl/tools

(will also improve consistency accros examples)